crates/hc_run_local_services/src/lib.rs

```diff
@@ -76,6 +76,28 @@ impl AOut {
 }
 
 impl HcRunLocalServices {
+    pub fn new(
+        bootstrap_address_path: Option<std::path::PathBuf>, 
+        bootstrap_interface: String,
+        bootstrap_port: u16,
+        disable_bootstrap: bool,
+        signal_address_path: Option<std::path::PathBuf>,
+        signal_interfaces: String,
+        signal_port: u16, 
+        disable_signal: bool,
+    ) -> Self {
+        Self {
+            bootstrap_address_path,
+            bootstrap_interface,
+            bootstrap_port,
+            disable_bootstrap,
+            signal_address_path,
+            signal_interfaces,
+            signal_port,
+            disable_signal,
+        }
+    }
```

Comment:
Guess we'd have to wrap the fields that have specific default values in newtypes, impl Default on them, and then let clap select its default values from those impls instead of specifying that in the attr. A bunch of annoying refactor busywork.